File: Modul-6/Soal-5/PRAK605-2410817220022-AmandaArvaSafaraya.c
#include <stdio.h>

int main() {
    int a;
    scanf("%d", &a);

    int matriksA[a][a];
    int matriksB[a][a];
    int hasil[a][a];

    printf("Matriks A\n");
    for (int i = 0; i < a; ++i) {
        for (int j = 0; j < a; ++j) {
            scanf("%d", &matriksA[i][j]); }
        }

    printf("Matriks B\n");
    for (int i = 0; i < a; ++i) {
        for (int j = 0; j < a; ++j) {
            scanf("%d", &matriksB[i][j]); }
        }

    for (int i = 0; i < a; ++i) {
        for (int j = 0; j < a; ++j) {
            hasil[i][j] = 0;
        }
    }

    for (int i = 0; i < a; ++i) {
        for (int j = 0; j < a; ++j) {
            for (int k = 0; k < a; ++k) {
                hasil[i][j] += matriksA[i][k] * matriksB[k][j];
            }
        }
    }

    printf("\nMatriks AXB\n");
    for(int i = 0; i < a; i++){
      for(int j = 0; j < a; j++){
        printf("%d ", hasil[i][j]);
      }
      printf("\n");
    }

    return 0;
}

File: Modul-6/Soal-5/PRAK605-2410817220022-AmandaArvaSafaraya.py
a = int(input())

print("Matriks A")
matriksA = [list(map(int, input().split())) for _ in range(a)]

print("Matriks B")
matriksB = [list(map(int, input().split())) for _ in range(a)]

hasil = [[0 for _ in range(a)] for _ in range(a)]
for i in range(a):
    for j in range(a):
        for k in range(a):
            hasil[i][j] += matriksA[i][k] * matriksB[k][j]
print("\nMatriks AXB")
for row in hasil:
    print(" ".join(map(str, row)))